nosnoc/nosnoc_opts.py

```diff
@@ -63,21 +63,20 @@ class NosnocOpts:
     equidistant_control_grid: bool = True  # NOTE: tested in test_ocp
 
     # IPOPT opts
-    opts_ipopt = dict()
-    opts_ipopt['print_time'] = 0
-    opts_ipopt['verbose'] = False
-    opts_ipopt['ipopt'] = dict()
-    opts_ipopt['ipopt']['sb'] = 'yes'
-    opts_ipopt['ipopt']['max_iter'] = 500
-    opts_ipopt['ipopt']['print_level'] = 0
-    opts_ipopt['ipopt']['bound_relax_factor'] = 0
+    opts_casadi_nlp = dict()
```

Comment:
Not sure I understand the need for this rename. Is this structure general to other casadi solvers?

Reply:
Yes.
```
    opts_casadi_nlp['print_time'] = 0
    opts_casadi_nlp['verbose'] = False
    opts_casadi_nlp['ipopt'] = dict()
```
Here the first 2 lines are casadi options.
The dict contains another dict for IPOPT options.
Thus the rename.